### ISSUE TITLE
Render crash when require native sharp module

### ISSUE THREAD

**Issue by topmat** [electron/electron#5641]
* Electron version: 0.37.8
* Operating system: OSX Yosemite 10.10.5

The following was the crash report of osx
https://gist.github.com/topmat/b43ea31bdc13c13f1b2508677c96b873

I'm try to use the "sharp" for resizing image. (https://github.com/lovell/sharp)
Before package, I'm using install_name_tool to change the dylib path.
If the window required the sharp module, the window will display white page. And the devtools will alert "DevTools was disconnected from the page".

**Comment by paulcbetts** [comment]
This looks like sharp just doesn't work right, or install_name_tool isn't working correctly. Does this module work with regular node.js?

**Comment by topmat** [comment]
I tried to use sharp in OSX and Windows. Windows is working.

For OSX. 
The "sharp" is working in our build environment before I use install_name_tool to change the dylib reference path.

After I patch(install_name_tools) all of the dylib that linked from sharp. Patch all of the related library path that external dylib. Use node to testing library is working. If I run sharp in the electron. I will get the crash when rendering. 

This list is library that I patched.
https://gist.github.com/topmat/b91a5714d88ac3d33d240a4f50e4adb7
